src/Services/FeatureChecker.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Supervisorg\Services;
+
+use Puzzle\Configuration;
+
+class FeatureChecker
+{
+    private
+        $configuration;
+
+    public function __construct(Configuration $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    public function isEnabled($name)
+    {
+        return $this->configuration->readRequired("features/$name/enabled");
```

Comment:
I think it's more secure to set these configuration entries optional.

```
 return $this->configuration->read("features/$name/enabled", true);
```

Btw, no BC break with this choice !